### PR TITLE
Pis layout work

### PR DESCRIPTION
**Pull request by justin-phxm**

### CODE REVIEW

File: app/src/renderer/code/App.tsx [hunk 1]
```diff
@@ -14,9 +13,9 @@ function App() {
   // 38, 55, 7
   return (
     <div className={currentAppState.darkMode ? "dark" : ""}>
-      <div className="h-screen w-screen overflow-x-hidden bg-light p-4 text-light dark:bg-dark dark:text-dark">
+      <div className="size-screen overflow-x-hidden bg-light p-4 text-light dark:bg-dark dark:text-dark">
```
Comment (ideen1) [inline app/src/renderer/code/App.tsx:16]: Please remove general App staying changes. These will eb handled in a different ticket

File: app/src/renderer/code/App.tsx [hunk 2]
```diff
@@ -26,8 +25,8 @@ function App() {
               </BrowserRouter>
             </div>
           </div>
-          <div className="flex flex-row flex-wrap gap-x-2 gap-y-2">
-            <div className="h-96 w-96 grow">
+          <div className="flex flex-row flex-wrap gap-2">
+            <div className="size-96 grow">
```
Comment (ideen1) [inline app/src/renderer/code/App.tsx:29]: Please remove general App staying changes. These will eb handled in a different ticket

File: app/src/renderer/code/components/atoms/FaultCard.tsx [hunk 3]
```diff
@@ -21,16 +19,16 @@ function FaultCard(props: FaultCardProps) {
       <div className="flex p-2">
         <div className="flex justify-self-start"></div>
         <div
-          className={`flex h-full w-full items-center justify-start border p-2 ${
-            severity == ISeverity.ERROR
+          className={`flex size-full items-center justify-start border p-2${
```
Comment (ideen1) [inline app/src/renderer/code/components/atoms/FaultCard.tsx:22]: Please remove general App staying changes. These will eb handled in a different ticket

File: app/src/renderer/code/components/containers/HeroContainer.tsx [hunk 4]
```diff
@@ -8,7 +6,7 @@ import GearParkBrakeComponent from "../molecules/HeroMolecules/GearParkBrakeComp
 function HeroContainer(props: any) {
   return (
     <>
-      <div className="grid h-full w-full grid-cols-1 gap-4 md:grid-cols-6">
+      <div className="grid size-full grid-cols-1 gap-4 md:grid-cols-6">
```
Comment (ideen1) [inline app/src/renderer/code/components/containers/HeroContainer.tsx:9]: Please remove general App styling changes. These will eb handled in a different ticket

File: app/src/renderer/code/components/containers/MapContainer.tsx [hunk 5]
```diff
@@ -17,7 +17,7 @@ function MapContainer(): JSX.Element {
   }, [carLocation]);
 
   return (
-    <div className="h-full w-full">
+    <div className="size-full">
```
Comment (ideen1) [inline app/src/renderer/code/components/containers/MapContainer.tsx:20]: Please remove general App styling changes. These will be handled in a different ticket

File: app/src/renderer/code/components/molecules/HeroMolecules/CarGraphicComponent.tsx [hunk 6]
```diff
@@ -1,6 +1,6 @@
-import React, { useState } from "react";
+/* eslint-disable react/no-unknown-property */
```
Comment (ideen1) [inline app/src/renderer/code/components/molecules/HeroMolecules/CarGraphicComponent.tsx:1]: @justin-phxm @brian-ngyn What are these ignores for? We should try to remove them if possible

**Review by ideen1 — CHANGES_REQUESTED** [review]
Please make the styling look identical to what is currently on main

**Review by IshaHaider — DISMISSED** [review]

**Review by ideen1 — APPROVED** [review]

**Review by brian-ngyn — APPROVED** [review]

**Pull request merged**